### ISSUE TITLE
Update Thorlabs PM100D driver to use "usbtmc" transport

### ISSUE THREAD

**Issue by jorisvr** [teamdiamond/qmi#42]
Our QMI driver for the Thorlabs PM100D currently uses pyvisa directly to access the power meter.
This should be changed so the driver uses the QMI transport mechanism. The usbtmc transport class will then map to the correct underlying implementation (pyvisa or python-usbtmc).

**Comment by jorisvr** [comment]
Done (commit e688603ee4faa9c7e9a208ae459410d30d91afc6).

The Thorlabs_PM100D now uses transport_usbtmc.
The driver thus requires a QMI transport descriptor ("usbtmc:vendorid=0x1313:productid=0x8078:serialnr=PMxxxx") instead of a VISA resource string.

The transport_usbtmc layer maps to PyVISA on Windows and to python-usbtmc on Linux.
We integrated the python-usbtmc module in QMI to avoid depending on an unmaintained github project.

**Comment by jorisvr** [comment]
We found that the Thorlabs PM100D is sometimes unresponsive when using python-usbtmc.
Typically, it causes timeout on the first interaction with the instrument, then recovers and responds normally. This happens in approximately 50% of the cases after opening the instrument.

The problem is probably caused by faulty DATA0/DATA1 toggle synchronization. We modified qmi.core.usbtmc to always call `set_configuration()` to work around this problem.
